MediaManager/Plugin.MediaManager.Abstractions/IMediaFile.cs

```diff
@@ -5,18 +5,9 @@
 
 namespace Plugin.MediaManager.Abstractions
 {
-    /// <summary>
-    /// Information about the mediafile
-    /// </summary>
-    /// <seealso cref="System.ComponentModel.INotifyPropertyChanged" />
     public interface IMediaFile : INotifyPropertyChanged
     {
         /// <summary>
-        /// A unique identifier for this media file
-        /// </summary>
-        Guid Id { get; set; }
```

Comment:
I agree on removing this, because an item is equal when the `Type` and `Url` match.